dev.hdos.HDOS.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "dev.hdos.HDOS",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "hdos",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=JAVA_HOME=/app/jre",
+        "--talk-name=org.freedesktop.Notifications",
+        "--filesystem=xdg-run/app/com.discordapp.Discord:create",
+        "--filesystem=~/hdos:create",
```

Comment:
I believe this is going to bypass the `--persist` line below. I'd just drop it at this point, `--persist=.` will do the trick.

Reply:
Alright, will do

Reply:
These files will still be accessible at `~/.var/app/dev.hdos.HDOS/hdos`. As I said, it's considered bad practice to place app-specific files directly in the top-level home directory.